### DISCUSSION TITLE
How to report diff for two CSV files?

### DISCUSSION THREAD

**Discussion by vitalyk-multinarity** [johnkerl/miller#1122] [Q&A] [answered]
I have two CSVs with the same header, some values may be different, for example:

```
kpi1,kpi2,kpi3,kpi4,...
1.23,5.67,2.22,3.45,4.44,...
```
I'd like to compare them and report a diff in a readable format, ideally with coloring.

(I already read about 'join' and reviewed many examples, but couldn't find proper CLI params...)
TIA,
Vitaly

**Comment by aborruso** [comment, marked as answer]
@vitalyk-multinarity I don't think Miller has what you are looking for, but maybe I'm wrong.

I know a specialized tool to do this: it's [daff](https://github.com/paulfitz/daff)

**Reply by vitalyk-multinarity** [reply]
Thank you, it's an amazing tool!
Vitaly
